test/e2e/data/shared/v1.7/metadata.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
```

Comment:
The folder for v1.5 should get removed (also for `data/infrastructure-docker/v1.5`)